### PR TITLE
Echo has same health as tank in ultimate

### LINKED ISSUE #228: Echo not having the same health as duplicated target (OW1 Ult)
When using her ult you don’t receive as much health as the target.

### PR DESCRIPTION
**Pull request by novalderya**
echo ult

fixes #228

### CODE REVIEW

File: src/ow1/heroes/echo.opy [hunk 1]
```diff
@@ -15,3 +29,19 @@ def initEcho():
...
+    @Condition eventPlayer.getHeroOfDuplication() in getTankHeroes()
+    
+    eventPlayer.hero_id_duplicated = heroID(eventPlayer.getHeroOfDuplication())
+    clearCustomHealth()
+    setCustomHealth(
+        health_pool[eventPlayer.hero_id_duplicated][0],
+        health_pool[eventPlayer.hero_id_duplicated][1],
+        health_pool[eventPlayer.hero_id_duplicated][2],
+    )
+    waitUntil(eventPlayer.isDuplicatingAHero() == false, 9999)
```
Comment (MaxwellJung) [inline src/ow1/heroes/echo.opy:47]: Why do you have to wait here?

File: src/constants/ow1_constants.opy [hunk 2]
```diff
@@ -164,6 +165,7 @@
 #!define OW1_ORISA_SUPERCHARGER_HEIGHT 1
 #!define OW1_ORISA_SUPERCHARGER_LINGER_TIME 1
 #!define OW1_ORISA_SUPERCHARGER_MAX_DURATION 15
+#!define OW1_ORISA_SHIELDS 0
```
Comment (MaxwellJung) [inline src/constants/ow1_constants.opy:168]: Can you reorder the constants so they're alphabetical for each hero?

File: src/ow1/heroes/echo.opy (hunk 1 above)
Comment (MaxwellJung) [inline src/ow1/heroes/echo.opy:47]: Add new line at EOF

```suggestion
    waitUntil(eventPlayer.isDuplicatingAHero() == false, 9999)

```

File: src/constants/ow1_constants.opy (hunk 2 above)
Comment (MaxwellJung) [reply src/constants/ow1_constants.opy:168]: I can also reorder it myself if you want to focus on something else.

File: src/ow1/heroes/echo.opy (hunk 1 above)
Comment (novalderya) [reply src/ow1/heroes/echo.opy]: It was for testing purpose, I forgot to remove it.

File: src/constants/ow1_constants.opy (hunk 2 above)
Comment (novalderya) [reply src/constants/ow1_constants.opy]: Done

**Review by MaxwellJung — APPROVED** [review]

**Pull request merged**